lib/src/models/special_item.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'package:wechat_assets_picker/wechat_assets_picker.dart';
+
+@immutable
+class SpecialItem {
+  SpecialItem({
+    required this.itemPosition,
+    this.itemBuilder,
+  });
+
+  /// Allow users set a special item in the picker with several positions.
+  /// 允许用户在选择器中添加一个自定义item，并指定位置
+  final SpecialItemPosition itemPosition;
+
+  /// The widget builder for the the special item.
+  /// 自定义item列表
+  final SpecialItemBuilder<AssetPathEntity>? itemBuilder;
+}
```

Comment:
Could `AssetPathEntity` be generic?

```suggestion
/// Allow users to set a special item in the picker grid with specified [position].
/// 允许用户在选择器中添加一个自定义item，并指定其位置。
@immutable
class SpecialItem<Path> {
  const SpecialItem({
    required this.builder,
    required this.position,
  });

  /// The widget builder for the the special item.
  /// 自定义item构建。
  final SpecialItemBuilder<Path>? builder;

  /// Define how the item will be positioned.
  /// 定义如何摆放item。
  final SpecialItemPosition position;
}
```